src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -5319,19 +5319,18 @@ private string BuildFontWeight(string memberValue)
 
 		private string BuildBrush(string memberValue)
 		{
-			var colorHelper = (INamedTypeSymbol)_metadataHelper.GetTypeByFullName(XamlConstants.Types.SolidColorBrushHelper);
+			var colors = (INamedTypeSymbol)_metadataHelper.GetTypeByFullName(XamlConstants.Types.Colors);
 
 			// This ensures that a memberValue "DarkGoldenRod" gets converted to colorName "DarkGoldenrod" (notice the lowercase 'r')
-			var colorName = colorHelper.GetProperties().FirstOrDefault(m => m.Name.Equals(memberValue, StringComparison.OrdinalIgnoreCase))?.Name;
+			var colorName = colors.GetProperties().FirstOrDefault(m => m.Name.Equals(memberValue, StringComparison.OrdinalIgnoreCase))?.Name;
 			if (colorName != null)
 			{
-				return "SolidColorBrushHelper." + colorName;
+				return $"new global::{XamlConstants.Types.SolidColorBrush}(global::{XamlConstants.Types.Colors}.{colorName})";
```

Comment:
This will likely be a performance issue. Maybe we should be using XamlBindingHelper to do the conversion, and still access SolidColorBrushHelper instead.

Reply:
@jeromelaban I don't see how this is different than before? Previously the getter was allocating a `SolidColorBrush` each time it's accessed. The new code should have the exact same performance characteristics as before, if I understand correctly.

Reply:
Ah that's right. I remembered an older version of the SolidColorBrush where instances were created only once. Thanks.